detect_os

```diff
@@ -60,8 +60,10 @@ install_deps()
         if ! command -v wget > /dev/null 2>&1; then
             if command -v yum > /dev/null 2>&1; then
                 yum install -y wget
-            else
+            elif command -v apt > /dev/null 2>&1; then
                 apt install -y wget
+            else
+                zypper install -y wget
```

Comment:
check for zypper as above.
Default is to bail out with error code and saying OS not known.